config.rb

```diff
@@ -81,3 +81,8 @@ def call(sprockets_asset)
 end
 
 sprockets.append_path File.join(root, "components")
+
+# https://middlemanapp.com/advanced/pretty-urls/
+activate :directory_indexes
+
+activate :asset_hash
```

Comment:
Right now, the configuration on frontend will serve images with `Last-Modified` and `ETag` headers and a `Cache-Control` header containing `max-age=0`, which means browsers with primed caches will send `If-Modified-Since` and `If-None-Match` headers and get 304 responses (this is a pretty good set-up if the files may change).

If we use asset hashing then we can set far-future `Cache-Control` and `Expires` headers (and maybe even use the exciting new `Cache-Control` `immutable` directive!). That doesn’t hold this up getting the product page changes live, but could you file a JIRA ticket on frontend to do this work?